### ISSUE TITLE
Basic Highlight Takes 52 Seconds to Complete

### ISSUE THREAD

**Issue by oleersoy** [johannestroeger/gulp-highlight#6]
Hi,

I'm not sure if the issue is with `gulp-highlight` or `highlightjs`.  I'm highlighting a few basic html blocks and the run takes 52 seconds.  Here's the tracing block:

``` js
      console.log("STARTINGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG");
      file.contents = new Buffer(highlight(file.contents.toString(), options), options.buffer);
      console.log("FINISHINGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG");
```

The rest of the explanation and repository to reproduce can be found here:

https://github.com/isagalaev/highlight.js/issues/1014

Ole

**Comment by oleersoy** [comment]
Please update the `highlightjs` dependency:

``` json
    "highlight.js": "~9.0.0",
```

That should fix it.

**Comment by johannestroeger** [comment]
hey - could you do a PR? tnx.

**Comment by oleersoy** [comment]
Hi, 

The markup that highlightjs generates has changed.  It is still correct though, per https://github.com/isagalaev/highlight.js/issues/1017.  @isagalaev suggested that the markup not be tested, since this is in `higlighjs`s control only.  Is it OK if I remove the test?

**Comment by oleersoy** [comment]
Actually on second thought why don't I just fix it.  I like that the test shows that highlightjs is at least working.

**Comment by oleersoy** [comment]
Just pushed the updated test.

**Comment by isagalaev** [comment]
> Actually on second thought why don't I just fix it.

Because it will break again! We change markup all the time, bugs getting fixed, features being added, that sort of stuff, you know :-) Since you can't know the reason for every change you will be bound to go and ask us every time, effectively working as a double continuous testing tool.

**Comment by isagalaev** [comment]
> I like that the test shows that highlightjs is at least working.

Practically speaking, I think a good simple test for that would be just checking for `<span` anywhere in the string.

**Comment by oleersoy** [comment]
@isagalaev Perfect - I really just wanted something that shows that highlightjs is working.  @johannestroeger if it's ok with you I'll update the test to reflect.

**Comment by oleersoy** [comment]
OK I'm just crossing my fingers.  I added Chai and updated the test to verify that the markup contains a span.

**Comment by oleersoy** [comment]
@johannestroeger any thoughts?

**Comment by johannestroeger** [comment]
released 2.0.0 with highlightjs 9.0.0 - thanks for your support!
